src/main/java/jag/DeleteCommand.java

```diff
@@ -0,0 +1,54 @@
+package jag;
+
+import java.io.IOException;
+
+/**
+ * Represents and instance of a DeleteCommand to delete a task
+ * from an instance of the TaskList and to call upon
+ * the Ui instance to display the appropriate message
+ */
+public class DeleteCommand extends Command {
+
+    /**
+     * Executes the command such that the required task gets deleted
+     * from the instance of TaskList, by calling upon Ui instance to
+     * get the index of the task to be deleted and displaying the right
+     * message
+     *
+     * @param tasks Represents instance of the TaskList to be accessed
+     *              such that the right task is deleted
+     * @param ui Represents the Ui instance for user interaction and to
+     *           break down the command input to get the right index
+     *           of the task to be deleted in the TaskList instance
+     * @param storage To update the output file storage after task
+     *                has been deleted
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
I like how you used plural form on TaskList as it represents a collection of Tasks!